lib/objects/objectsInMemClient.js

```diff
@@ -1,587 +1,12 @@
-        client.emit('delObject', id, options, callback);
-    };
-
-    this.delObjectAsync = function (id, options) {
-        if (!client) return Promise.reject('No client');
-        return new Promise((resolve, reject) => {
-            this.delObject(id, options, err => {
-                if (err) {
-                    reject(err);
-                } else {
-                    resolve();
-                }
-            });
-        });
-    };
-
-    this.getObject = function (id, options, callback) {
-        if (typeof options === 'function') {
-            callback = options;
-            options = null;
-        }
-        if (!client) return;
-        if (!callback) {
-            return new Promise((resolve, reject) => {
-                this.getObject(id, options, (err, obj) => {
-                    if (err) {
-                        reject(err);
-                    } else {
-                        resolve(obj);
-                    }
-                });
-            });
-        } else {
-            client.emit('getObject', id, options, callback);
-        }
-    };
-
-    this.getObjectAsync = function (id, options) {
-        if (!client) return Promise.reject('No client');
-        return new Promise((resolve, reject) => {
-            this.getObject(id, options, (err, obj) => {
-                if (err) {
-                    reject(err);
-                } else {
-                    resolve(obj);
-                }
-            });
-        });
-    };
-
-    this.findObject = function (idOrName, type, options, callback) {
-        if (typeof options === 'function') {
-            callback = options;
-            options = null;
-        }
-        if (!callback) {
-            return new Promise((resolve, reject) => {
-                this.findObject(idOrName, type, options, (err, obj) => {
-                    if (err) {
-                        reject(err);
-                    } else {
-                        resolve(obj);
-                    }
-                });
-            });
-        } else {
-            if (!client) return;
-            client.emit('findObject', idOrName, type, options, callback);
-        }
-    };
-
-    this.findObjectAsync = function (idOrName, type, options) {
-        if (!client) return Promise.reject('No client');
-        return new Promise((resolve, reject) => {
-            this.findObject(idOrName, type, options, (err, obj) => {
-                if (err) {
-                    reject(err);
-                } else {
-                    resolve(obj);
-                }
-            });
-        });
-    };
-
-    this.destroyDB = function (options, callback) {
-        if (typeof options === 'function') {
-            callback = options;
-            options = null;
-        }
-        if (!client) return;
-        client.emit('destroyDB', options, callback);
-    };
+let Objects = null;
+try {
+    // @ts-ignore
+    Objects = require('./objectsInRedis');
```

Comment:
Why are you ignoring the error? It says `cannot find module` (and I can't either).
If this is really not an error, extend the comment: `// @ts-ignore why are we ignoring this?`

Reply:
// @ts-ignore normally class comes from objects-redis package, but in special cases is can exist locally

Basically exists directly for dev purposes and other special installation cases, but normally it comes from objects-redis package (obfruscated)